packages/sdk/client/src/diagnostics/diagnostics.ts

```diff
@@ -91,7 +93,10 @@ export const createDiagnostics = async (client: Client, options: DiagnosticOptio
     const stream = client.services.services.LoggingService!.queryMetrics({});
     const trigger = new Trigger<Metrics>();
     stream?.subscribe(async (metrics) => trigger.wake(metrics!));
-    data.metrics = await trigger.wait();
+    data.metrics = await asyncTimeout(trigger.wait(), DEFAULT_DIAGNOSTICS_TIMEOUT).catch((err) => {
+      log.warn('Metics take to long to query.');
```

Comment:
log.catch(err) or log.warn(err.message).
i.e., no need to "reword" error messages like this (the original error message is good and tells the user about the timeout).
OR just let throw since oclifs error handler should catch it.